src/transport/tls/listen.rs

```diff
@@ -117,13 +117,13 @@ impl<L: HasConfig, G> Listen<L, G> {
         <Fut as IntoFuture>::Future: Send,
         Self: GetOriginalDst + Send + 'static,
     {
-        self.listen_and_fold_inner(std::u64::MAX, initial, f)
+        self.listen_and_fold_inner(std::usize::MAX, initial, f)
     }
 
     #[cfg(test)]
     pub fn listen_and_fold_n<T, F, Fut>(
         self,
-        connection_limit: u64,
+        connection_limit: usize,
```

Comment:
Is the change from `u64` to `usize` related to the fix here? Just curious.

Reply:
yeah, buffered takes usize